src/components/FilterPanel.jsx

```diff
@@ -73,9 +81,44 @@ export default function FilterPanel({ onClose }) {
             <Typography>{name}</Typography>
           </AccordionSummary>
 
-          <AccordionDetails sx={{ m: 1 }}>{filter}</AccordionDetails>
+          <AccordionDetails sx={{ m: 1 }}>{filter.comp}</AccordionDetails>
         </Accordion>
       ))}
+
+      <Stack direction="row" spacing={1} sx={{ m: 2 }}>
+        {filtersChanged ? (
+          <WarningAmberOutlinedIcon color="warning" />
+        ) : (
+          <CheckCircleOutlineIcon color="success" />
+        )}
+        <Typography>
+          {filtersChanged
+            ? "Your changes haven't yet been applied."
```

Comment:
shortend `have not` to `haven't` to avoid the text to stack when on desktop

Reply:
nit: this should use single quotes.

Reply:
Makes sense, you could also shorten it to `Your changes are not yet applied`, which seems to be closer in length to the other message (look how much screen width each of these take up on your screen):

Your changes haven't yet been applied.
Your filters are currently applied.
Your changes are not yet applied.